### PR TITLE
Update setup.md to be more explicit. 

### PR DESCRIPTION
**Pull request by omenking**
* I can understand why we have a package.json file with very specific versions though we can encourage users to install these packages themselves so they are using the latest versions to avoid security vulnerabilities but also still provide an example so if they are having issues they can just copy ours with the fixed versions.
* We should set explicitly the host and port for devServer. Users are generally likely going to need to change these settings for their environment and this will save them time trying to hunt down these options.
* For cloud IDEs such a Cloud9 you need to use port `0.0.0.0`. This will save users significant time debugging connectivity issues.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by nikhil-dabhade** [comment]
@brene @sammartinez This is in the Js getting started guide. Can you take a look and validate the changes?

**Comment by sammartinez** [comment]
> @brene @sammartinez This is in the Js getting started guide. Can you take a look and validate the changes?

Verifying now

File: docs/start/getting-started/fragments/vanillajs/setup.md [hunk 1]
```diff
@@ -123,6 +156,8 @@ module.exports = {
 };
 ```
 
+By default `webpack-dev-server` will start on port `8080`. You may need to change devServer host to `0.0.0.0` if you are running your web-server in a Cloud IDE such as AWS Cloud9.
```
Comment (brene) [inline docs/start/getting-started/fragments/vanillajs/setup.md]: I think this is too much detail for a getting started. My thought: If people are aware of Cloud IDEs they probably will know how to setup webpack. @sammartinez - thoughts?

Comment (brene) [reply docs/start/getting-started/fragments/vanillajs/setup.md]: Chatted with Sam offline. This seems fine. I'll resolve this comment.

Comment (sammartinez) [reply docs/start/getting-started/fragments/vanillajs/setup.md]: This should be fine, but we may want to recommend locking down to your IP so this isnt open to the world

File: docs/start/getting-started/fragments/vanillajs/setup.md [hunk 2]
```diff
@@ -18,7 +18,37 @@ The app directory structure should be:
...
-Add the following to the `package.json` file:
+Intitalize a new `package.json` file using the following command:
+```
+npm init -y
+```
+
+We'll need to install dev dependencies 
+
+```
+npm i \
```
[the remark this answers is not in the document]
Comment (sammartinez) [reply docs/start/getting-started/fragments/vanillajs/setup.md]: @omenking can you verify this works in Windows?

**Comment by brene** [comment]
@omenking - let me know if you have any questions regarding my comments.

**Comment by brene** [comment]
@omenking - just checking in if you can address our comments

File: docs/start/getting-started/fragments/vanillajs/setup.md (hunk 1 above)
Comment (brene) [inline docs/start/getting-started/fragments/vanillajs/setup.md:159]: Can you add a ">" around this. e.g.
> By default `webpack-dev-server` will start on port `8080`. You may need to change devServer host to `0.0.0.0` if you are running your web-server in a Cloud IDE such as AWS Cloud9.

**Pull request closed without merging**